imports/wasi_snapshot_preview1/fs.go

```diff
@@ -1349,11 +1360,11 @@ var pathCreateDirectory = newHostFunc(
 func pathCreateDirectoryFn(_ context.Context, mod api.Module, params []uint64) Errno {
 	fsc := mod.(*wasm.CallContext).Sys.FS()
 
-	dirFD := uint32(params[0])
```

Comment:
some places we accidentally used dirFD which was a test property, in the main tree, where the actual param is called `fd`.